src/utils.js

```diff
@@ -52,39 +82,56 @@ function getComponentNames(basePaths) {
     return names.sort();
 }
 
-/**
- * @param gmeConfig
- * @param logger
- * @param callback
- * @returns {*}
- */
-function copySvgDirsAndRegenerateSVGList(gmeConfig, logger, callback) {
-    var deferred = Q.defer(),
-        svgAssetDir = path.join(__dirname, 'client', 'assets', 'DecoratorSVG');
+function getSVGMap(gmeConfig, logger, callback) {
```

Comment:
The amount of extra svgs is user specific and not bounded. Don't see why it should block?